x-pack/plugins/alerts/server/invalidate_pending_api_keys/task.ts

```diff
@@ -214,9 +217,9 @@ async function invalidateApiKeys(
             `Failed to cleanup api key "${apiKeyObj.attributes.apiKeyId}". Error: ${err.message}`
```

Comment:
Should we update this to say `Failed to delete invalidated API key` to be more descriptive?